lib/cbus_elixir_web/controllers/page_controller.ex

```diff
@@ -1,11 +1,26 @@
 defmodule CbusElixirWeb.PageController do
+  import Ecto.Query, only: [from: 2]
```

Comment:
This seems like a smell that model behavior is in the wrong place.

Reply:
Yeah, L9:16 should go into PageView as a function that is called from either the controller or the template.

Reply:
Sure...have to figure out how.

Reply:
Okay, I moved the function to the PageView and commented out the function on the controller. Now it cannot see the function. Aren't there already wired up to talk to each other?

Reply:
Helpers is not where it goes either, IMO - this should be done in `CbusElixir.App.Meeting`. Something like:
```elixir
  def next_meeting_for(today) do
    Repo.one!(from m in Meeting,
      where: m.date > type(^today, Ecto.Date),
      order_by: [asc: m.date],
      preload: [:speakers],
      limit: 1)
  end
```

Parts of this query should likely be extracted to separate functions on `Meeting` composed with `|>`, but that's a separate issue.

Reply:
I need to do some more reading on this - "Programming Phoenix" suggests keeping query-building in the models but interactions with `Repo` in the controller (Chapter 7)